learn/chapter1/4-textures.md

```diff
@@ -122,7 +122,7 @@ Another solution, and probably a good one, is to use an image-loading library th
 
 ImageSharp is a very popular image loading library by SixLabors that is able to load most popular file formats and is easy to integrate in your project(s). You can add ImageSharp to your project from Nuget.
 
-For the following section on textures, we'll use an [image of a wooden crate](https://learnopengl.com/img/textures/container.jpg).
+For the following section on textures, we'll use an [image of a wooden crate](textures/container.jpg).
```

Comment:
Can you make sure that this is included in the build? Download DocFX if you haven't already, and then run `docfx build` and see if the textures subfolder pops up in the output. You'll need to setup your directories as described in the README.

Alternatively, you could just put everything in the same folder. Yeah, it'd get messy but it's easier.

Reply:
Done, can confirm that the textures subfolder ended up in the output as-is. No changes needed.

Reply:
Awesome